guide/english/css/fonts/index.md

```diff
@@ -32,13 +25,15 @@ In addition to specifying common fonts that are found on most operating systems,
 Then you can use the font that you have imported (in this example, `Lobster`) in your code as normal.
 
 ### Font style
-The `font-style` property can be used to specify italic text.
+The `font-style` property can be used to specify the text style.
 
 This property has 3 values:
 
 * normal - Text shown normally
 * italic - Text shown in *italic*
-* oblique - Text shown leaning
+* oblique - Text shown slanted
+
+If a font does not have an italic version, the oblique version may be substituted, and vice versa; if neither version exists, the style will be simulated.
```

Comment:
Referencing this [MDN article](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style)

> When the specified style is not available, both italic and oblique faces are simulated by artificially sloping the glyphs of the regular face (use font-synthesis to control this behavior).

but rereading now, I agree that it could be clearer. Will edit.